components/get_more_page_data.tsx

```diff
@@ -10,11 +10,14 @@ export default async function GetMorePageData(dataParams) {
     if (response.error === "No access!") {
       Router.push('/')
     } else {
-      if (response.length < 10) {
+      if (response.data.length < 10) {
         dataParams.setHasMore(false)
       }
       dataParams.setDataLength(dataParams.dataLength + 10)
-      dataParams.setData((bookings) => [...bookings, ...response])
+      dataParams.setData((bookings) => [...bookings, ...response.data])
```

Comment:
dataParams -> getDataParams

Reply:
We need a comment at the beginning explaining what parameters does this function receives